src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -72,33 +74,38 @@
     public static final String PARSE_LIST_SUCCESS_MESSAGE_FORMAT = "category: %s\n";
     public static final String PARSE_GET_SUCCESS_MESSAGE_FORMAT = "itemName: %s\nproperty: %s\n";
 
-    public static final String INVALID_COMMAND_MESSAGE_STRING = "Invalid command, please try again.";
     public static final String PARSE_SUCCESS_MESSAGE_STRING = "Parsed successful.\n";
+    public static final String INVALID_COMMAND_MESSAGE_STRING = "Invalid command, please try again.";
 
-
+    private static final Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
 
     /**
      * Parses the User input line. Checks the user input line against the basic command format
      * and extracts the command word which is the first word in the user input line. After
      * extraction of the command word, pass the remaining user input line arguments to the
      * respective cases depending on the command word.
      *
-     * @param userInputLine The user input Line
-     * @param list The itemContainer used to prepare the command
-     * @return A string indicating parse success or failure
+     * @param userInputLine The user input Line.
+     * @param list          The itemContainer used to prepare the command.
+     * @return Command object depending on the command type.
+     * @throws IllegalFormatException   If user input line does not match the respective command format.
+     * @throws ItemNotExistException    If item name not found in the container.
```

Comment:
I think its better not to leave full stop here, according to https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/documentation.html#javadoc

Reply:
Noted